### PR TITLE
Mol 15870 - Add OneService theme

### PR DESCRIPTION
**Pull request by ryhuz**
**Changes**
Added OneService theme - includes colours and text styles

- [delete] branch

**Changelog entry**

- Added OneService theme - includes colours and text styles

**Additional information**

- You may refer to this [MOL-15870](https://jira.ship.gov.sg/browse/MOL-15870)

### CODE REVIEW

File: src/spec/color-spec/oneservice-color-set.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        },
+        Blue: {
+            Text: "#064D70", // Validation Text
+            Icon: " #066391",
+            Border: "#84B7DB",
+            Background: "#E8F4FA", // Validation Background
+        },
+    } as ValidationTypes,
+    Shadow: {
+        Accent: "rgba(87, 169, 255, 0.5)",
```
Comment (qroll) [inline src/spec/color-spec/oneservice-color-set.ts:68]: should be `rgba(45, 121, 144, 0.5)`

File: stories/getting-started/themes.mdx [hunk 2]
```diff
@@ -25,6 +25,9 @@ import { MyLegacyTheme } from "@lifesg/react-design-system/theme";
 
 // The RBS theme
 import { RBSTheme } from "@lifesg/react-design-system/theme";
+
+// The OneService theme
+import { OneService } from "@lifesg/react-design-system/theme";
```
Comment (qroll) [inline stories/getting-started/themes.mdx:30]:
```suggestion
import { OneServiceTheme } from "@lifesg/react-design-system/theme";
```

File: stories/color/oneservice-color.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { Meta, Unstyled } from "@storybook/blocks";
+import { OneServiceColorSet } from "src/spec/color-spec/oneservice-color-set";
+import { Text } from "src/text";
+import { PreviewBox, Secondary, Title } from "../storybook-common";
+import { ColorPalette } from "./doc-color-palette";
+
+<Meta title="General/Color/OS2.0 Color" />
+
+<Title>OS2.0 Colors</Title>
```
Comment (qroll) [inline stories/color/oneservice-color.mdx:9]: can we call this OneService also 🌝

**Review by qroll — CHANGES_REQUESTED** [review]

**Comment by qroll** [comment]
noticed in the Figma specs that a separate color was declared for hyperlinks - how is your app planning to tackle that? as links are fixed to the Secondary theme color

File: stories/color/oneservice-color.mdx (hunk 3 above)
Comment (ryhuz) [reply stories/color/oneservice-color.mdx]: updated

File: stories/getting-started/themes.mdx (hunk 2 above)
Comment (ryhuz) [reply stories/getting-started/themes.mdx]: updated

File: src/spec/color-spec/oneservice-color-set.ts (hunk 1 above)
Comment (ryhuz) [reply src/spec/color-spec/oneservice-color-set.ts]: updated

**Comment by ryhuz** [comment]
> noticed in the Figma specs that a separate color was declared for hyperlinks - how is your app planning to tackle that? as links are fixed to the Primary/Secondary theme color

will be handled separately with UX input

**Review by qroll — APPROVED** [review]

**Pull request merged**